unit-tests/py/rspy/devices.py

```diff
@@ -1,24 +1,39 @@
 # License: Apache 2.0. See LICENSE file in root directory.
 # Copyright(c) 2021 Intel Corporation. All Rights Reserved.
 
+from rspy import log
+
+# We need both pyrealsense2 and acroname. We can work without acroname, but
+# without rs no devices at all will be returned.
 try:
-    from rspy import acroname
+    import pyrealsense2 as rs
+    log.d( rs )
+    try:
+        from rspy import acroname
+    except ModuleNotFoundError:
+        # Error should have already been printed
+        # We assume there's no brainstem library, meaning no acroname either
+        acroname = None
 except ModuleNotFoundError:
-    # Error should have already been printed
-    # We assume there's no brainstem library, meaning no acroname either
+    log.w( 'No pyrealsense2 library is available! Running as if no cameras available...' )
```

Comment:
Why not wrapping this try catch around 
 "import pyrealsense2 as rs"
like you did for 
"from rspy import acroname"?

Reply:
Not sure what you mean.. let's talk